### PR TITLE
Ft/support dsl testplan

### PR DESCRIPTION
**Pull request by bsels**
At the request of @pdawyndt, support DSL testplan as input for the judge.

### CODE REVIEW

File: tested/languages/javascript/templates/run.mako [hunk 1]
```diff
@@ -91,12 +91,11 @@ try {
         ${ctx.before}
         % for testcase in ctx.testcases:
             writeSeparator();
+            <% testcase: _TestcaseArguments %>\
```
Comment (niknetniko) [inline tested/languages/javascript/templates/run.mako:94]: Why was this moved outside the `try-catch`?

Comment (bsels) [reply tested/languages/javascript/templates/run.mako:94]: Because when you declare a variable in the `try-catch` it'sn't visible in the other `try-catches`

**Review by niknetniko — APPROVED** [review]
